### DISCUSSION TITLE
Tips/Code/Best Practices for Dealing with Pre/Post COVID in models

### DISCUSSION THREAD

**Discussion by katedamme** [now-i-know-my-abcd/docs#43] [Stats & Code Help]
Hello Everyone!

I am curious if anyone has battled with the pre/post covid issues in their longitudinal data? I am not particularly interested in the impact of COVID as the main question but want to make sure that I can detect and adjust for the potential impact of COVID on my measures of interest.

Are people generally leaning toward a cross lag analyses that crossreferences interview dates/COVID dates? 

Does anyone have code that they are using to code pre/post COVID? I have some R code that I used but it's clunky and I think could be improved if folks have suggestions! 

`ABCD_weekly_activity_scales_yr2[order(as.Date(ABCD_weekly_activity_scales_yr2$fit_ss_protocol_startdate, format="%m/%d/%Y")),]->ABCD_weekly_activity_scales_yr2
as.Date(ABCD_weekly_activity_scales_yr2$fit_ss_protocol_startdate, format="%m/%d/%y") < "2/03/20"->ABCD_weekly_activity_scales_yr2$PostCovid_InterviewDate
unique (ABCD_weekly_activity_scales_yr2$fit_ss_protocol_startdate)
unique (ABCD_weekly_activity_scales_yr2$fit_ss_protocol_startdate)->unique_dates
order(as.Date(unique_datesdf, format="%m/%d/%y"))
data_frame(unique_dates)->unique_datesdf
PPSEXTINT_nda%>%filter(PPSEXTINT_nda$eventname=="3_year_follow_up_y_arm_1")->PPSEXTINT_nda_3yr

PPSEXTINT_nda_3yr[order(as.Date(PPSEXTINT_nda_3yr$interview_date, format="%m/%d/%y")),]->PPSEXTINT_nda_3yr
as.Date(PPSEXTINT_nda_3yr$interview_date) < "2/03/20"->PPSEXTINT_nda_3yr$PostCovid_InterviewDate`

Thanks!

**Comment by niamhmacsweeney** [comment]
I've been thinking about this too Kate but haven't yet worked with it in the longitudinal ABCD data. Would be keen to hear how others are approaching this too!

**Comment by msf51** [comment]
Hi! Curious if you resolved this issue, I am thinking through the best way to account for COVID in my longitudinal ABCD study as well.

**Comment by wkthompson123** [comment]
I'm not sure if there are best practices, I think it depends on what the research question is: pre-COVID is generally pretty well defined but "post-COVID" is not. I think this is a tricky topic, and it also depends on what is meant by "the effect of COVID". Is it something specific (e.g., school closures, being infected or being in a family with severe illness, media coverage, public policies), or is it just a general "COVID period" difference in measures? 

There are also the RECOVER resources coming down the pike this year or next.

**Reply by msf51** [reply]
Thanks for replying that is helpful. I am trying to control for a general "COVID period" difference in measures. I am looking at family conflict, prosocial behavior, and school risk and protective factors, which I imagine would be most affected by school closures. 

We came up with making a categorical variable to represent study visit dates occurring pre-COVID (before March 15, 2020), early-COVID (between March 15, 2020 and August 16, 2021), and late-COVID (after August 16, 2021). We used those dates because it looks like those were the dates between which data were gathered by ABCD on local COVID-related policies, but since we are just controlling for the time period we didn't want to get deeper into the dates of policy changes themselves. I feel good about the first date, but does the August 16th, 2021 date sound like a reasonable approximation of the transition to "late COVID"? Is there a reason ABCD selected that end date when examining policy changes? Thanks!
